content/secure-remote-access/README.md

```diff
@@ -0,0 +1,56 @@
+# Secure Remote Access
+
+## Overview
+Instead of using qidi link to access your printer, this guide servces to help you 
+setup a VPN mesh network that allows you to access your printer using various klipper
+apps for mobile devices using your trusted network as opposed to using Qidi's. 
+
+## Prerequisites
+* Comfortable with SSH access to your printer and executing commands
+* Have an free tailscale account - https://tailscale.com/signup/
+
+## 1. Setup Tailscale VPN Account
+Follow the onboarding guidelines from tailscale to create and register your first device, preferabbly your phone since you are replacing qidi link! Instructions are [here](https://tailscale.com/kb/1017/install). Once you see your phone listed in the devices, move on to the next step!
+
+## 2. Disable qidi link
+Since we don't need this service running anymore, let's scavange back those resources! SSH to your printer and then we will disable the qidi link service with the following command:
+
+```
+sudo systemctl disable --now QIDILink-client.service
+```
+
+This command will stop the service and also disable it from starting up! 
+
+## 3. Install Tailscale VPN On The Printer
+Following the documentation from tailscale [here](https://tailscale.com/kb/1041/install-debian-buster), we will install the tailscale client on the printer with the following command:
```

Comment:
tailscale provides a [oneline install script on their download page](https://tailscale.com/download/linux) which would adapt if tailscale deprecates this particular way of installing tailscale, or changes the keys, or anything like that. while `curl | bash` isn't a very nice practice in general, I'd suggest using the faster and solution provided from a trusted party (tailscale) over having to go over a number of commands that may or may not stop working down the line as debian buster gets even more ancient than it already is

Reply:
Yeah, also fair and rightful feedback. I did a quick audit of the script oneliner script and it currently does account for crusty debian 10: https://github.com/tailscale/tailscale/blob/main/scripts/installer.sh#L79-L80
 
 I'll update this to be more concise and to leverage the one liner.

Reply:
@vega-d Hows this look? Glad you called this out actually! I've always just used the wiki to install to the platform. Didn't even know that they had an install script!
https://github.com/qidi-community/Plus4-Wiki/pull/31/commits/e1feadef34038046334bf78487ab0ce248a5f557